### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by eudondian**
-  A new project with webpack based on the webpack exercise already completed.
-  An index.html file was created and  HTML markup was written here. 
-  index.html file was set as a template using the HTML Webpack Plugin.
-  An index.js file was created and an array of some simple to-do tasks (array of objects) was set. 
-  Each task object contains three keys:
-  description [string].
-  completed [bool].
-  index: [number].
-  A style.css was created and rules were set for the To-Do List. CSS was loaded by Webpack Style/CSS Loader

Add ReadME.md file

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (arslanbisharat) [inline src/index.js:1]: - [ ] Let’s not disable linter rules these are here to help us write a good price of code

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    index: 2,
+  },
+  {
+    description: 'build a form',
+    completed: false,
+    index: 3,
+  },
+];
+
+// eslint-disable-next-line no-unused-vars
```
Comment (arslanbisharat) [inline src/index.js:20]: - [ ] Let’s not disable linter rules these are here to help us write a good price of code

**Review by arslanbisharat — APPROVED** [review]
Hi @eudondian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Dear @eudondian, 
I would like to apologise for the mistake. As per the guidelines you have on GiTHub you can disable linter rules.
Please submit a review request and the next Code Reviewer will approve it. 

Thank you

**Review by vechicin — APPROVED** [review]
# Hi @eudondian ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job setting up Webpack

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**